functests/functest.py

```diff
@@ -191,9 +215,11 @@ def run_test(self, tcount, test, default_normalize=None, check_path=None):
 
         if normalization_func is not None:
             actual_output = normalization_func(actual_output)
+            # print('ACTUAL NORMALISED:', actual_output, file=sys.stderr)
```

Comment:
Is it worth removing commented-out code or wrapping it, so the output is displayed if a flag is set?

Reply:
It's a mistake - ta for flagging